.env.example

```diff
@@ -53,4 +53,5 @@ MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
 
 VONAGE_KEY=
 VONAGE_SECRET=
-VONAGE_SMS_FROM=
\ No newline at end of file
+VONAGE_SMS_FROM=
+MOBILE_MAX_ATTEMPTS=3
```

Comment:
@MaxRomagnoli instead of default value here, it should be a variable in `config/app.php` or one of the config files:

```
'mobile_attempts_left' => env('MOBILE_MAX_ATTEMPTS', 3),
```

And then no need for default value in `.env.example`, and we can use `config('app.mobile_attempts_left')` everywhere